File: word.py
s = input()
upper = 0
lower = 0

for i in range(len(s)):
    if int(chr(s)) >= 65 and 90 <= int(chr(s)):
        upper+=1
    else:
        lower+=1

if upper == lower:
    print(s.lower())

if upper > lower:
    print(s.upper())

if lower > upper:
    print(s.lower())

File: word.java
import java.util.Scanner;

public class word{
	
	static Scanner in = new Scanner(System.in);

	public static void main(String [] args){
	
		String s = in.nextLine();
		int upper = 0;	
		int lower = 0;	

		
		for(int i = 0; i < s.length(); i++){
			char c = s.charAt(i);

			if(c >= 65 && c <= 90){
				upper++;
			}
			else{
				lower++;
			}
		}


		if(upper > lower){
			System.out.println(s.toUpperCase());
		
		}
		else{
			System.out.println(s.toLowerCase());
		}
	}
}
